### ISSUE TITLE
City Planning - Public Lands

### ISSUE THREAD

**Issue by blancoagostina** [decentraland/proposals#43]
**Name:** Ephemeral Lands

**Purpose:** Free content creation

**Description:**
Inspired on ‘Lennon Wall’ from Prague, I’d like to propose the creation of public Lands where everybody who visit Decentraland will be able to edit or modify, without the need to be a Land owner. 

![prague](https://user-images.githubusercontent.com/11165157/28800112-b665a1bc-7621-11e7-82c5-65516c0bb639.png)

**Concept**
This wall continuously undergoes change and everyday shows a new face to the people who visit it. Today represents a symbol of global ideals such as love and peace. 

In my opinion it would be nice to have this 'ephemeral' places in Decentralnd designated for the free creation of content, such as its deconstruction along the time.

**Reddit style** 
Recently, for April fools' day, Reddit allowed the community to draw things on a canvas letting users to modify a tile/pixel color per every 5 minutes. It quickly became a community-driven labor with coordinated efforts to build, defend and rebuild different images.

You can see the timelapse [here](https://www.youtube.com/watch?v=XnRCZK3KjUY)

![rplace](https://user-images.githubusercontent.com/11165157/28800662-0ab65502-7624-11e7-86d5-eba2722599d0.jpg)

**Comment by EmmaMakhno** [comment]
i mentioned a script to allow land owners to turn this feature on voluntarily so any dcl citizen would be able to graffiti their walls. also it would be nice to have this feature turned on in some public areas that are traditionally covered in grafitti. train tunnels and the like

**Comment by blancoagostina** [comment]
Yes! It'd be amazing to see graffitis all over Decentraland :) Also I'd love to see 3D creations!

**Comment by maraoz** [comment]
Love this!
